crates/swc_ecma_codegen/tests/test262-min/664b0da1dd015106.js

```diff
@@ -1 +1 @@
-a["b"]="c";a["if"]="if";a["*"]="d";a["\u0EB3"]="e";a[""]="f"
```

Comment:
Here the same, esbuild prefer to print `\u0EB3`, terser - real unicode character